library/functions.lisp

```diff
@@ -44,6 +46,12 @@
       (lisp :a (str item) (cl:format cl:t "~A: ~A~%" str item))
       Unit))
 
+  (declare print ((Into :a String) => :a -> :a))
+  (define (print item)
+      "Print the String representation of an item to `cl:*standard-output*` and return the item."
```

Comment:
nit:

```
"Print the string representation of `item` to the standard output, and return the item."
```

Maybe it should actually just return `Unit`? What do you think?

Reply:
I made it return `Unit`, and added a function `and-return`, so that, for debugging purposes, one can write `(and-return print x)` for example. I also used `and-return` to implement `vector:singleton`.